src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -56,3 +57,8 @@ def get(self, svc_id):
             'osd_metadata': mgr.get_metadata('osd', svc_id),
             'histogram': histogram,
         }
+
+    @RESTController.Resource('POST')
```

Comment:
Now that #22318 was merged you can change this to: `@RESTController.Resource('POST', query_params=['deep'])`

Reply:
Done